README.md

```diff
@@ -1,21 +1,24 @@
 # Capstone_Appointment_back_end
 
-> This application is developed using Ruby on Rails(RoR) CRUD capabilities. It allows users to create motorcycles and appointments to purchase and post them in a web interface. This app uses RoR, RSPec, Capybara, Devise, and CanCanCan.
+> This application is developed using Ruby on Rails(RoR) CRUD capabilities. It allows users to create motorcycles and appointments to purchase and post them in a web interface.
 
 ## Built With
 
 - Ruby On Rails (RoR)
 - PostegreSQL
 - RSpec
-- Capybara
-- Devise
-- CanCanCan
-- Bullet
+- Rswag
+- jwt
+- Bcrypt
+- rack-cors and
+- image-processing
 
 ## Live Demo (if available)
 
 [Live Demo Link]()
-
+[API Documentation](https://motocycle-booking.herokuapp.com/api-docs/)
+[Front-end link](https://final-capstone-project-udqc.vercel.app/splash_page)
+ 
```

Comment:
- [ ] On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

 ## Tips and extra help

Go [here] (https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪